service/controller/v2/resources/vaultcrt/delete.go

```diff
@@ -46,7 +46,7 @@ func (r *Resource) NewDeletePatch(ctx context.Context, obj, currentState, desire
 	return patch, nil
 }
 
-func (r *Resource) newDeleteChange(ctx context.Context, obj, currentState, desiredState interface{}) (interface{}, error) {
+func (r *Resource) newDeleteChange(ctx context.Context, currentState interface{}) (interface{}, error) {
```

Comment:
I think this is not good. It is also not necessary anymore. We disabled the noparam check. Reason is to keep the interfaces consistent for automation reasons. Code generation will be easier. Fixing interfaces and maintaining our operators will be more easily automatable when we keep the interfaces aligned. We just had that with the go modules move where we used some scripts to adapt to the recent microerror advances.

Reply:
Fixed